runner/master_log.py

```diff
@@ -156,6 +168,20 @@ def messages(self, test_id: str = "") -> list[str]:
 
         return messages
 
+    def messages_past_question(self, test_id: str) -> list[str]:
+        messages = []
+        getting_all_messages = False
+        for event in self.log:
+            if event.type in [EventType.SEND_MESSAGE, EventType.RESPONSE_MESSAGE, EventType.SEND_FILL, EventType.RESPONSE_FILL]:
+                if test_id == event.test_id or getting_all_messages:
+                    sender = "Test" if event.type == EventType.SEND_MESSAGE else "System" if event.type == EventType.SEND_FILL else "Agent"
+                    messages.append(f"{sender} ({event.timestamp}): {event.data['message']}")
+
+                    if event.data["is_question"]:
+                        getting_all_messages = True
+
+        return messages
```

Comment:
I have trouble understanding this method. Why is `getting_all_messages` set only when it finds a question? It's starting from the very first logged event, so it could be any question.

Reply:
Ah I see. The question is when the callback is registered. Alright 👌🏾

Reply:
This method is almost the same as `master_log.messages(test_id)`. Could it work to rename it to `master_log.test_readable_log(test_id, is_callable)`?